tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -475,13 +476,12 @@ def test_positive_update_name(self):
             with self.subTest(new_name):
                 ContentView.filter.update({
                     'content-view-id': self.content_view['id'],
-                    'name': cvf_name,
+                    'id': cvf['filter-id'],
                     'new-name': new_name,
-                    'organization-id': self.org['id'],
```

Comment:
Agreed, it's not specific for org-id, we may think about adding such test. My point was it will catch overall org removal, as i can't remember issue when org-id was removed, basically if someone touches orgs - all 3 options org-id org-name and org-label become removed :)